### PR TITLE
Add noroadsleft keymap for KC60

### PR DESCRIPTION
**Pull request by noroadsleft**
Finally decided to add my keymap to the QMK repo.

### CODE REVIEW

File: keyboards/kc60/keymaps/noroadsleft/config.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline keyboards/kc60/keymaps/noroadsleft/config.h]: You should have `#pragma once` at the beginning

File: keyboards/kc60/keymaps/noroadsleft/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// https://github.com/qmk/qmk_firmware/issues/3448#issuecomment-406636125
```
Comment (drashna) [inline keyboards/kc60/keymaps/noroadsleft/rules.mk]: Makefiles use `#` for comments, IIRC

Comment (noroadsleft) [reply keyboards/kc60/keymaps/noroadsleft/rules.mk]: Huh, it's never warned me that this is wrong when I've compiled. But you're right.

**Comment by noroadsleft** [comment]
Discovered a couple of bugs in my macros today. Trying to fix...

**Comment by drashna** [comment]
Just let me know when this is ready.

**Comment by noroadsleft** [comment]
Finally remembered to test the fixes for the bugged macros today. This is good to go.

**Pull request merged**